src/base/datastore/persistedState.js

```diff
@@ -1,9 +1,11 @@
 import createPersistedState from 'vuex-persistedstate'
+import { throttle } from 'quasar'
 
 export default createPersistedState({
   paths: [
     'i18n.locale',
     'auth.push.intention',
     'auth.push.token',
   ],
+  subscriber: store => handler => store.subscribe(throttle(handler, 5000)),
```

Comment:
I think we might forget some values this way. For example when the locale changes twice within 5 seconds, it would only persist the first value.
`debounce` might be a bit better, as it would always persist the last value. It fails when the user quits the app before the debounce time is over.

Reply:
Yes, good point. I originally used `debounce`, but switched to `throttle` to catch the case of leaving the page. In freegle I recently added handling to use the `beforeunload` event to persist changes before leaving the page... but would need to do a kind of custom `debounce` for that. Maybe I do that...

Reply:
Did it! `lodash.debounce` has a nice `.flush()` method that I can call in the unload handler to flush any calls that are waiting... fits in nicely.